pkg/ops/iso.go

```diff
@@ -31,33 +32,49 @@ func GenISO(name, src, dst string, i schema.ISO) func(ctx context.Context) error
 			overlay = i.DataPath
 		}
 
+		// We are assuming StepCopyCloudConfig has already run, putting it the config in "dst"
```

Comment:
umm this is bad. Should we check if there is a dst/config.yaml before continuing?

Reply:
I will add a check but that doesn't change the fact that we still assume the other step to have run (temporal dependency). Since steps have dependencies on other steps by definition, maybe that's ok?

Reply:
done

Reply:
It may be that geniso maybe should not copy the config? Maybe we need another steps that actually copies the config only and that it's triggered by the dag dependency? Anyway, can wait